src/components/App.js

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from './Button'
+import Header from './Header'
+import Prompt from './Prompt'
+
+function App() {
+  const prompts = [
```

Comment:
Maybe you could create a json file containing this array, to shorten this.

`App.js`
``` 
import prompts from './data/prompts.json'
```

`data/prompts.json`
```
[
  "Big Book",
  "Bought Reently",
  ...
]
```